en/ai/README.md

```diff
@@ -1,25 +1,25 @@
 # AI functionality in JabRef
 
-Since version 6, JabRef has AI-functionality build in.
+Since version 6, JabRef has AI functionality built in.
 
 - AI can generate a summary of a research paper
-- One can also chat with papers using a "smart" AI assistant
+- You can also chat with papers using a "smart" AI assistant
```

Comment:
Different style. I opted for "one", because the other JabRef documentation does not have any "you".

But I think, it is OK. Consistency can be done later.